client/src/styles/GameThread.css

```diff
@@ -0,0 +1,18 @@
+.Title {
```

Comment:
This should be a lowercase "title" class to be consistent

Reply:
yup!